uwb/example/lib/game_widgets/score.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:flutter/material.dart';
+
+/// The [Score] class that generates the score.
+class Score extends StatelessWidget {
+  /// The [Score] constructor.
+  const Score({Key? key, required this.score, required this.gameHasStarted})
+      : super(key: key);
+
+  /// Int variable to keep track of the current score.
```

Comment:
Same as with the documentation on the `brickFieldList` field of the `BrickField` class:

- Don't mention information that is already present in the signature;
- The field is marked `final` which means it won't change, so make sure the documentation reflects this.

 ```suggestion
  /// The [score] to be displayed.
```